charts/jaeger-operator/values.yaml

```diff
@@ -23,6 +24,11 @@ rbac:
   pspEnabled: false
   clusterRole: false
 
+service:
+  type: ClusterIP
+  # Specify a specific node port when type is NodePort
+    # nodePort: 32500
```

Comment:
we can align the indents :thinking:

Reply:
I'd aligned it as such, because the service yaml file requires `.Values.service.type.nodePort` to pass in the custom NodePort value.